config/base.yaml

```diff
@@ -10,6 +10,12 @@ email:
   noreply_email_name: "UniCS"
   email_verification_email_subj: "Verify email"
   password_reset_email_subj: "Reset password"
+  token_lifetime: 108000 # 30 hours
 app_url: "auth.unicsmcr.com"
 data_policy_url: "https://drive.google.com/file/d/1wMcJbfEhIp9FjdNbyom4RVUoTH4xc0OB/view"
 soft_max_team_members: 4
+auth:
+  user_token_lifetime: 108000 # 30 hours
```

Comment:
With this addition, we'll have three different token lifetimes defined in the config file. I think this could lead to confusion especially between the `auth_token_lifetime` and `auth -> user_token_lifetime` and which value is used. Maybe adding some comments explaining the difference would help here?

Reply:
Yeah I want to tidy up the config once auth v2 is done. `auth -> user_token_lifetime` would replace `auth_token_lifetime`. API v1 and the frontend handlers still use it so I can't remove `auth_token_lifetime` yet.

Reply:
Ah I see. In the future would all the properties inside the `auth` section be moved to the top level of the config?

Reply:
The opposite. I've just added the `auth` sub-config (that's probably not a word) and auth related config would get moved into `auth`